plugins/terminal.koplugin/main.lua

```diff
@@ -83,6 +83,17 @@ local T = require("ffi/util").template
 
 local CHUNK_SIZE = 80 * 40 -- max. nb of read bytes (reduce this, if taps are not detected)
 
+-- This should be moved inside `platform/android/luajit-launcher/assets/android.lua`
+-- once we now it works as expected
+if Device:isAndroid() and not os.getenv("SHELL") and os.execute() == nil then
```

Comment:
It still does not make sense to me to do it conditionally and end up with different implementation with completely different behaviors, one of them incompatible with how `os.execute` is supposed to work (again, supporting `cmd … >redirect`, like on other platforms).

Reply:
So once moved to `android.lua`, that variant of the function is always going to be used? No `not os.getenv("SHELL") and os.execute() == nil` conditional?

For testing, why not just use `if Device:isAndroid() then`?